tempodb/backend/azure/azure_helpers.go

```diff
@@ -7,12 +7,17 @@ import (
 	"strings"
 	"time"
 
+	"github.com/Azure/azure-pipeline-go/pipeline"
 	blob "github.com/Azure/azure-storage-blob-go/azblob"
+	"github.com/cristalhq/hedgedhttp"
 )
 
-const maxRetries = 3
+const (
+	maxRetries        = 1
+	maxHedgedRequests = 2
```

Comment:
The `gcs` and `s3` package have similar properties (`uptoHedgedRequests`). Do we want to keep these separate or combine them somewhere? It might be nice to have them somewhere central so we are sure they stay aligned. But moving them out of the package also removes a lot of the surrounding context though...

Reply:
i think it's fine to leave them internal even though they have the same value. one day they may become independent settings anyway.